### PR TITLE
Improve formatting of round() examples

### PR DESCRIPTION
**Pull request by drewnoakes**
- Align outputs
- Move explanatory text from source to comment

### CODE REVIEW

**Comment by Jak-MS** [comment]
@shsagir 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/kusto/query/roundfunction.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ that the `round()` function rounds a number to a specific number of digits while
 ## Examples
 
 ```kusto
-round(2.15, 1)                   // 2.2
-round(2.15) (which is the same as round(2.15, 0))                   // 2
-round(-50.55, -2)                   // -100
-round(21.5, -1)                   // 20
+round(2.15, 1)      // 2.2
+round(2.15)         // 2     same as round(2.15, 0)
```
Comment (shsagir) [inline data-explorer/kusto/query/roundfunction.md:34]:
```suggestion
round(2.15)         // 2 - same as round(2.15, 0)
```

**Comment by shsagir** [comment]
@drewnoakes Thanks for your feedback. We've submitted a documentation change based on your suggestion, and the updated article should be live by tomorrow. Please let us know if there's anything more we can do for you. To follow up on this issue, leave us a comment with an @mention.

#sign-off

File: data-explorer/kusto/query/roundfunction.md [hunk 2]
```diff
@@ -30,8 +30,8 @@ that the `round()` function rounds a number to a specific number of digits while
 ## Examples
 
 ```kusto
-round(2.15, 1)                   // 2.2
-round(2.15) (which is the same as round(2.15, 0))                   // 2
-round(-50.55, -2)                   // -100
-round(21.5, -1)                   // 20
+round(2.15, 1)      // 2.2
+round(2.15)         // 2 - same as round(2.15, 0)
```
Comment (drewnoakes) [inline data-explorer/kusto/query/roundfunction.md:34]: This now resembles a mathematical subtraction.

**Comment by Court72** [comment]
#hold-off while outstanding PR comments are resolved.

File: data-explorer/kusto/query/roundfunction.md (hunk 2 above)
Comment (shsagir) [inline data-explorer/kusto/query/roundfunction.md:34]:
```suggestion
round(2.15)         // 2 // equivalent to round(2.15, 0)
```

Comment (shsagir) [reply data-explorer/kusto/query/roundfunction.md]: Good point .. updated

Comment (shsagir) [inline data-explorer/kusto/query/roundfunction.md]:
```suggestion
round(2.15)         // 2 // same as round(2.15, 0)
```

**Comment by shsagir** [comment]
#sign-off

**Pull request merged**